jira-feature-collector/src/main/java/com/capitalone/dashboard/datafactory/jira/JiraDataFactoryImpl.java

```diff
@@ -254,8 +255,20 @@ public String setQuery(String query) {
 				}
 			} catch (Exception e) {
```

Comment:
@amitmawkin , I take that back again... I was right - Jira's RestClientException does not also throw or handle the following exception:  ```java.net.UnknownHostException```.  It is thus also uncatchable - I am nearly forced to throw it on my method implementation, myself.  You can see this in the failed build Travis.ci build for the above-noted commit.

However, I was able to handle it myself in a different way - I am now directly validating if a Jira host provided is an available namespace before ever attempting to resolve any query promises.  Please review my code at the following commit/PR:  https://github.com/capitalone/Hygieia/pull/606/commits/60af6cb2ccdf926a8318a1e26938b7b4e7b94114